mantidproject/mantid
# Fix LARMOR MultiPeriod system tests

**Description of work.**
This fixes the tests by removing the conflicting test file from the unit tests.

**To test:**
 - Check that the following tests do not fail:
```
SANS2DMultiPeriodAddFiles.LARMORMultiPeriodAddEventFiles
SANS2DMultiPeriodAddFilesTest_V2.LARMORMultiPeriodAddEventFilesTest_V2
```
 - Check no other SANS/LARMOR tests fail.

_No associated issue_

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
